docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -56,6 +56,10 @@ Property Name                                      Required   Description
                                                               default the location is ``/hbase``
 ``phoenix.config.resources``                       No         Comma-separated list of configuration files (e.g. ``hbase-site.xml``) to use for
                                                               connection properties.  These files must exist on the machines running Trino.
+``phoenix.max-scans-per-split``                    No         Maximum number of HBase scans allowed to be grouped into a single split. Default is 20.
```

Comment:
WDYT about - `Maximum number of HBase scans that can be performed in a single split`.